ledger/accountdb.go

```diff
@@ -2097,6 +2115,56 @@ func performTxTailTableMigration(ctx context.Context, tx *sql.Tx, blockDb db.Acc
 	return err
 }
 
+func performOnlineRoundParamsTailMigration(ctx context.Context, tx *sql.Tx, blockDb db.Accessor, newDatabase bool, initProto protocol.ConsensusVersion) (err error) {
+	totals, err := accountsTotals(tx, false)
+	if err != nil {
+		return err
+	}
+	if newDatabase {
+		onlineRoundParams := []ledgercore.OnlineRoundParamsData{
+			{
+				OnlineSupply:    totals.Online.Money.Raw,
+				RewardsLevel:    totals.RewardsLevel,
+				CurrentProtocol: initProto,
+			},
+		}
+
+		err = accountsPutOnlineRoundParams(tx, onlineRoundParams, 0)
+		return err
+	}
+
+	err = blockDb.Atomic(func(ctx context.Context, blockTx *sql.Tx) error {
+		rnd, err := accountsRound(tx)
+		if err != nil {
+			return nil
+		}
+
+		hdr, err := blockGetHdr(blockTx, rnd)
+		if err != nil {
+			return nil
+		}
+
+		var data ledgercore.OnlineRoundParamsData
+		err = tx.QueryRowContext(ctx, "SELECT online, rewardslevel FROM accounttotals").Scan(&data.OnlineSupply, &data.RewardsLevel)
+		if err != nil {
+			return err
+		}
```

Comment:
as discussed, the history table cannot be recreated from a single row of accounttotals